### PR TITLE
Added an information about how to set a default value

### PR DESCRIPTION
**Pull request by VitaliyBoyko**
## Purpose of this pull request

This PR provides information about how to set the default value of custom address attribute on the checkout address page.

## Affected DevDocs pages
* https://devdocs.magento.com/guides/v2.1/howdoi/checkout/checkout_new_field.html
* https://devdocs.magento.com/guides/v2.2/howdoi/checkout/checkout_new_field.html
* https://devdocs.magento.com/guides/v2.3/howdoi/checkout/checkout_new_field.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4631) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4631) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/howdoi/checkout/checkout_new_field.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ $customField = [
     'filterBy' => null,
     'customEntry' => null,
     'visible' => true,
+    // value is used to set a default value of the attribute
+    'value' => ''
```
Comment (rogyar) [inline guides/v2.1/howdoi/checkout/checkout_new_field.md]: Hi @VitaliyBoyko. Could you set the inline commend on the same line as the value, please? Also, adjust the comment slightly by adding the "value field" clarification
```suggestion
    'value' => '' // value field is used to set a default value of the attribute
```

Comment (VitaliyBoyko) [reply guides/v2.1/howdoi/checkout/checkout_new_field.md]: @rogyar
Done.
Thanks!

**Pull request merged**